doc/release/upcoming_changes/26268.improvement.rst

```diff
@@ -0,0 +1 @@
+Scalars and 0D arrays are disallowed for `numpy.nonzero` and `numpy.ndarray.nonzero`.
```

Comment:
Should start with a `*` and be named `expired.rst` (IIRC)

Reply:
Done!